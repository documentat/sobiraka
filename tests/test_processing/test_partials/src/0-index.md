# One include

{% include 'part.md' %}
