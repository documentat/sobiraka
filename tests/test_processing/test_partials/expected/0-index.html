<h1>One include</h1>

<p>This text is from a partial.</p>
